src/modules/billing/mappers/transaction.js

```diff
@@ -96,7 +97,8 @@ const dbToModelMapper = createMapper()
   .map('volume').to('volume', volume => volume ? parseFloat(volume) : null)
```

Comment:
If I remember correctly, Russell recently fed back that the it's OK for volumes to be zero by default. With that in mind, should it be `.map('volume').to('volume', volume => volume ? parseFloat(volume) : 0)` as opposed to `.map('volume').to('volume', volume => volume ? parseFloat(volume) : null)`?

Ignore if not

Reply:
I think it's OK for the volume to be null for min charge transactions.  However I agree the logic looks suspect.  

I might expect it to be:
```
 volume => isNull(volume) ? null : parseFloat(volume)
```
otherwise it won't handle zero properly.